### DISCUSSION TITLE
Stuck in teaching

### DISCUSSION THREAD

**Discussion by GuenterWarth** [jomjol/AI-on-the-edge-device#2245] [General]
Hello, hopefully someone of you can help out. My ESP32 stuck in teaching.
5x flashed (ereased before)
different SD Card tested.
[log_2023-03-26.zip](https://github.com/jomjol/AI-on-the-edge-device/files/11071108/log_2023-03-26.zip)

What did I wrong.

**Comment by caco3** [comment]
Check https://jomjol.github.io/AI-on-the-edge-device-docs/StatusLED-BlinkCodes/ for the flashing code.

Looking on the log it looks ok.
It completed at least 2 rounds. enable info ort debug mode for more details.

The `Neg. Rate` indicates that you should adjust the `Pre Value`, see menu `settings -> set previous value`


Note that there is no such thing as a `teaching` mode!
